tools/cli/workflowCommands.go

```diff
@@ -629,6 +629,11 @@ func QueryWorkflowUsingStackTrace(c *cli.Context) {
 	queryWorkflowHelper(c, "__stack_trace")
 }
 
+// QueryWorkflowUsingQueryTypes list all query types of the workflow using __query_types as query type
+func QueryWorkflowUsingQueryTypes(c *cli.Context) {
+	queryWorkflowHelper(c, "__query_types")
```

Comment:
This feature on the client side was not yet released and is not yet supported by java client.
I'm not sure what is the best way to communicate this.

Reply:
I think that's fine to have CLI doing more than you can do programmatically.

Reply:
it is, but it's also implying [something] will work.

what's the failure message when this reaches a worker that doesn't support the query?  it should probably be something more descriptive than a default one.